datasets/varen/dataset.json

```diff
@@ -13,166 +13,167 @@
-          "naamVaartuig": {
-            "type": "string",
-            "description": "Naam van vaartuig."
-          },
-          "naamKlantKvk": {
-            "type": "string",
-            "description": "Naam van klant volgens KvK"
-          },
-          "ligplaatsSegment": {
-            "type": "string",
-            "description": "Categorie van lengte van vaartuigen"
-          },
-          "kenmerkVergunning": {
-            "type": "string",
-            "description": "Kenmerk van ligplaastvergunning"
-          },
-          "datumEindeVergunning": {
-            "type": "string",
-            "description": "Einde van verleende vergunning"
-          },
-          "idLigplaats": {
-            "type": "string",
-            "description": "Id veld (niet uniek)",
-            "provenance": "var_ligplaats_id"
-          },
-          "verwerkingsdatum": {
-            "type": "string",
-            "format": "date"
-          },
-          "gbdBuurt": {
-            "type": "string",
-            "description": "Unieke identificatie van het object",
-            "relation": "gebieden:buurten",
-            "uri": "https://www.amsterdam.nl/stelselpedia/gebieden-index/catalogus/buurt"
-          },
-          "locatieAdres": {
-            "type": "string",
-            "description": "Locatie beschrijving"
-          },
-          "geometrie": {
-            "$ref": "https://geojson.org/schema/Geometry.json",
-            "description": "Geometrie in RD (epsg:28992)"
+          "$schema": "http://json-schema.org/draft-07/schema#",
+          "type": "object",
+          "additionalProperties": false,
+          "required": [
+              "schema",
+              "id"
+          ],
+          "mainGeometry": "geometrie",
+          "display": "id",
+          "properties": {
+              "schema": {
+                  "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+              },
+              "id": {
+                  "type": "string",
+                  "description": "Uniek identificerend kenmerk van het record",
+                  "provenance": "var_ligplaats_id"
+              },
+              "naamVaartuig": {
+                  "type": "string",
+                  "description": "Naam van vaartuig."
+              },
+              "naamKlantKvk": {
+                  "type": "string",
+                  "description": "Naam van klant volgens KvK"
+              },
+              "ligplaatsSegment": {
+                  "type": "string",
+                  "description": "Categorie van lengte van vaartuigen"
+              },
+              "kenmerkVergunning": {
+                  "type": "string",
+                  "description": "Kenmerk van ligplaastvergunning"
+              },
+              "datumEindeVergunning": {
+                  "type": "string",
+                  "description": "Einde van verleende vergunning"
+              },
+              "idNietUniek": {
```

Comment:
Dit maakt het schema incompatible met alle historische data, wat zou betekenen dat het naar versie 2.0.0 zou moeten gaan. Het is misschien makkelijker om dit veld zijn oude naam te laten behouden, `idLigplaats`.

Reply:
dit is een kleien dataset bestaande uit 374 records. Elke avond lezen we deze dataset opnieuw. Dus de vorige versie zal in de historie laag wordt gehistoriseerd.  De kolom `idNietUniek` is aanleiding voor deze wijziging.